pagopa/io-app-design-system
# [IOBP-80] Add `light` and `lightFilled` icons

## Short description
This PR adds `light` and `lightFilled` icons.

<img src="https://github.com/pagopa/io-app-design-system/assets/6160324/f8a14986-567f-4d6a-8cd7-f5c39885b5ad" width="250" />

## List of changes proposed in this pull request
- Added `IconLight.svg` asset and `IconLight.tsx` component
- Added `IconLightFilled.svg` asset and `IconLightFilled.tsx` component
- Added `light` and `lightFilled` in `IOIcons`.

## How to test
On the example app, go to Icons and check that the new icons are displayed correctly.
